File: 0217 contains-duplicate/solution.ts
function containsDuplicate(nums: number[]): boolean {
    const s = new Set();
    for(let i=0;i<nums.length;i++){
        if(s.has(nums[i])){
            return true;
        }
        s.add(nums[i])
    }
    return false;
};

File: 0217 contains-duplicate/solution.rs
use std::collections::HashSet;

impl Solution {
    pub fn contains_duplicate(nums: Vec<i32>) -> bool {
        let mut set = HashSet::new();
        for i in 0..nums.len() {
            if(set.contains(&nums[i])) {
                return true;
            }
            set.insert(nums[i]);
        }
        return false;
    }
}
